pylzy/lzy/env/explorer/packages.py

```diff
@@ -27,6 +27,7 @@ class LocalDistribution(LocalPackage):
 class PypiDistribution(BasePackage):
     version: str
     pypi_index_url: str
+    platform_present: bool
```

Comment:
name is a bit misleading, which platform we mean? we mean here "server-supported" platform, so may be `platform_supported` or smth like that?